doc/classes/EditorSceneImporterAssetImport.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
It would be good to edit the commit log to be more explicit about what it does. Some of the logs of past squashed commits also don't seem relevant. Adding details on what is actually implemented, what works and what doesn't, would be nice.

Reply:
This comment was meant for the whole commit, not just this specific file. I just commented here as the PR review feature no longer allows adding commit-wide comments.